pkg/plugins/ifaces.go

```diff
@@ -93,6 +93,7 @@ type Client interface {
 	backend.AdmissionHandler
 	backend.CallResourceHandler
 	backend.CollectMetricsHandler
+	backend.QueryMigrationHandler
```

Comment:
😬 -- lets not add more to this unless we really need to.  Can we use the admission handler for grpc, and then maybe add utility go functions in the SDK that make it specifically query focused?

Reply:
I commented on this in the other PR: https://github.com/grafana/grafana-plugin-sdk-go/pull/1029#issuecomment-2225070285